hedera-node/hedera-app-spi/src/main/java/com/hedera/node/app/spi/TransactionHandler.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hedera.node.app.spi;
+
+/** Sub-types define methods for handling transactions. */
+public interface TransactionHandler {}
```

Comment:
De we really want to use marker interfaces instead of annotations? I never got the real usage for such interfaces since Annotations are part of the language